tests/integration/test_dataset.py

```diff
@@ -149,21 +149,42 @@ def test_data_row_export(dataset, image_url):
     assert set(result) == ids
 
 
-def test_dataset_export_v2(dataset, image_url):
+def test_dataset_export_v2(export_v2_test_helpers, dataset, image_url):
     n_data_rows = 5
-    ids = set()
+    datarows = set()
     for _ in range(n_data_rows):
-        ids.add(dataset.create_data_row(row_data=image_url))
+        datarows.add(dataset.create_data_row(row_data=image_url))
 
     time.sleep(10)
```

Comment:
Might be good to use `wait_for_data_row_processing` function instead of doing a sleep here. This function will wait until data rows are finished processing media attributes. Same goes for `test_dataset_export_v2_datarow_list`